test/BaselineOutput/Common/FeatureContribution/AveragePerceptronBinary.tsv

```diff
@@ -4,14 +4,15 @@
 #@   col=X2VBuffer:R4:1-4
 #@   col=X3Important:R4:5
 #@   col=Label:R4:6
-#@   col=Features:R4:7-12
-#@   col=Features:R4:13-18
-#@   col=FeatureContributions:R4:19-24
-#@   col=FeatureContributions:R4:25-30
-#@   col=FeatureContributions:R4:31-36
-#@   col=FeatureContributions:R4:37-42
+#@   col=Label:BL:7
```

Comment:
Any reason to have two label columns?

Reply:
Ditto

---
In reply to: [263191537](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263191537) [](ancestors = 263191537)